src/config/cloudinary.js

```diff
@@ -0,0 +1,17 @@
+const cloudinary = require('cloudinary').v2;
+
+cloudinary.config({
+  cloud_name: process.env.CLOUDINARY_CLOUD_NAME,
+  api_key: process.env.CLOUDINARY_API_KEY,
+  api_secret: process.env.CLOUDINARY_API_SECRET,
+});
+
+const DEFAULT_IMAGE_URL =
```

Comment:
These should be in an .env but you can fix that later. 

Just imagine your application becomes huge and you need to scale it, you would want image sources at the CDN location that is serving your web application.